types/ros2d/index.d.ts

```diff
@@ -0,0 +1,400 @@
+// Type definitions for ros2d.js 0.9.0
+// Project: http://wiki.ros.org/ros2djs
```

Comment:
Can you point to https://github.com/RobotWebTools/ros2djs instead of the current page? It's not a big deal, but that's the general approach we use.